packages/react-components/src/components/Caption.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { IPersona, Persona, Stack, useTheme, PersonaSize, Text, IStackTokens, mergeStyles } from '@fluentui/react';
+import React from 'react';
+import { _pxToRem } from '@internal/acs-ui-common';
+import { _FileUploadCardsStrings } from './FileUploadCards';
+import { OnRenderAvatarCallback } from '../types';
+
+/**
+ * @internal
+ * _Caption Component Props.
+ */
+export interface _CaptionProps {
+  userId: string;
+  userName: string;
```

Comment:
for consistency consider `displayName` instead of `userName` as we use displayName in other components (participantListItem, tpyingIndicator)